content/en/events/2022/kcsna/schedule.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Schedule
+type: docs
+weight: 3
+---
+
+
+## Schedule
+
```

Comment:
Similar comment as the one for the social page.

Reply:
removed headers from all of them 👍

Reply:
Thanks!